### DISCUSSION TITLE
WiFi Power Save or ?

### DISCUSSION THREAD

**Discussion by jazzjohn27** [earlephilhower/arduino-pico#1121] [Q&A] [answered]
Sometimes it takes a few tries for a browser on the LAN to access the Pico W's webserver. 

Other times, when that problem doesn't exist, the first access is slower (  2-3 seconds) , but then subsequent accesses are faster (less than 1 second).  If I wait 2 minutes, then the first access is slower again.

I read that the WiFi module goes into a power save mode by default. Could this be causing connectivity problems?

**Comment by earlephilhower** [comment, marked as answer]
@brabl2 gave a PR https://github.com/earlephilhower/arduino-pico/pull/1046 that disabled the low power mode by default.  Since the WiFi core is a binary blob it's hard to know if it's going to sleep or if what you're seeing is due to a higher level thing (i.e. Ethernet ARP timeout, etc.).
